WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Blaze/DashboardBlazeCampaignCardCell.swift

```diff
@@ -87,6 +87,12 @@ final class DashboardBlazeCampaignCardCell: DashboardCollectionViewCell {
         BlazeFlowCoordinator.presentBlaze(in: presentingViewController, source: .dashboardCard, blog: blog)
     }
 
+    override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        super.traitCollectionDidChange(previousTraitCollection)
+
+        createCampaignButton.setTitleColor(UIColor.primary, for: .normal)
+    }
+
```

Comment:
Do we need this?

Reply:
It looked like it wasn't updating and was too dark yesterday evening. I double-checked with a color meter, and no, it does update, and it's not needed. Removed. Something off with my color perception 😁

Reply:
🙈